Installation/Mapping/XxllncCaseTypeToZGWZaakType.json

```diff
@@ -2,27 +2,30 @@
-    "vertrouwelijkheidaanduiding": "instance.properties.designation_of_confidentiality",
+    "vertrouwelijkheidaanduiding": "{{ instance.properties.designation_of_confidentiality|lower }}",
     "verlengingMogelijk": "instance.properties.extension",
     "publicatieIndicatie": "instance.properties.publication",
     "omschrijving": "instance.title",
     "opschortingEnAanhoudingMogelijk": "instance.properties.suspension",
-    "statustypen": "[{% for phase in instance.phases %}{% if not loop.first %},{% endif %}{{ map('https://development.zaaksysteem.nl/mapping/xxllnc.XxllncPhaseToZGWStatusType.mapping.json', phase)|json_encode }}{% endfor %}]",
-    "informatieobjecttypen": "[{% set index=0 %}{% for phase in instance.phases %}{% for field in phase.fields %}{% if field.type == 'file' %}{% if index != 0 %},{% endif %}{{ map('https://development.zaaksysteem.nl/mapping/xxllnc.XxllncFieldToZGWInformatieObjectType.mapping.json', field)|json_encode }}{% set index=index+1 %}{% endif %}{% endfor %}{% endfor %}]",
-    "eigenschappen": "[{% set index=0 %}{% for phase in instance.phases %}{% for field in phase.fields %}{% if field.type != 'file' %}{% if index != 0 %},{% endif %}{{ map('https://development.zaaksysteem.nl/mapping/xxllnc.XxllncFieldToZGWEigenschap.mapping.json', field)|json_encode }}{% set index=index+1 %}{% endif %}{% endfor %}{% endfor %}]",
-    "roltypen": "[{% set index=0 %}{% for phase in instance.phases %}{% if phase.route.role|default %}{% if index != 0 %},{% endif %}{{ map('https://development.zaaksysteem.nl/mapping/xxllnc.XxllncRoleToZGWRolType.mapping.json', phase.route.role)|json_encode }}{% set index=index+1 %}{% endif %}{% endfor %}]",
-    "resultaattypen": "[{% if instance.results|default %}{% for result in instance.results %}{% if not loop.first %},{% endif %}{{ map('https://development.zaaksysteem.nl/mapping/xxllnc.XxllncResultToZGWResultaatType.mapping.json', result)|json_encode }}{% endfor %}{% endif %}]",
+    "statustypen": "[{% for phase in instance.phases %}{% set phase = phase|merge({'_catalogus': _catalogus, '_zaaktypeIdentificatie': instance.legacy.zaaktype_id}) %}{% if not loop.first %},{% endif %}{{ map('https://development.zaaksysteem.nl/mapping/xxllnc.XxllncPhaseToZGWStatusType.mapping.json', phase)|json_encode }}{% endfor %}]",
+    "informatieobjecttypen": "[{% set index=0 %}{% for phase in instance.phases %}{% for field in phase.fields %}{% if field.type == 'file' %}{% set field = field|merge({'_catalogus': _catalogus, '_zaaktypeIdentificatie': instance.legacy.zaaktype_id}) %}{% if index != 0 %},{% endif %}{{ map('https://development.zaaksysteem.nl/mapping/xxllnc.XxllncFieldToZGWInformatieObjectType.mapping.json', field)|json_encode }}{% set index=index+1 %}{% endif %}{% endfor %}{% endfor %}]",
```

Comment:
No, because there is a condition that can block the mapping from running. Using loop.first would in some cases result in a comma in front of everything.